common/sgx/verifier.c

```diff
@@ -33,11 +34,10 @@ typedef oe_mutex oe_mutex_t;
 #endif
 
 static const oe_uuid_t _local_uuid = {OE_FORMAT_UUID_SGX_LOCAL_ATTESTATION};
-static const oe_uuid_t _ecdsa_uuid = {OE_FORMAT_UUID_SGX_ECDSA_P256};
-static const oe_uuid_t _ecdsa_report_uuid = {
-    OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT};
-static const oe_uuid_t _ecdsa_quote_uuid = {
-    OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE};
+static const oe_uuid_t _ecdsa_uuid = {OE_FORMAT_UUID_SGX_ECDSA};
+static const oe_uuid_t _legacy_report_uuid = {
```

Comment:
Suggest making name match define (notably, to include "remote"):
```suggestion
static const oe_uuid_t _or_format_uuid_legacy_report_remote = {
```

Reply:
Thanks!